includes/template-library/manager.php

```diff
@@ -66,23 +66,6 @@ public function __construct() {
 	public function add_actions() {
 		add_action( 'elementor/ajax/register_actions', [ $this, 'register_ajax_actions' ] );
 		add_action( 'wp_ajax_elementor_library_direct_actions', [ $this, 'handle_direct_actions' ] );
-
-		// TODO: bc since 2.3.0
-		add_action( 'wp_ajax_elementor_update_templates', function() {
```

Comment:
Check

Reply:
Tried to check with Elementor 2.2.7, Template Library wouldn't even load (JS error). Not sure how to proceed. 
I guess for safety we can revert this removal and keep this code.